packages/repo-tools/src/generate-configs.mts

```diff
@@ -215,6 +215,7 @@ async function main(): Promise<void> {
     ];
     const flatExtends: string[] = [];
     const flatConfig: FlatConfig.Config = {
+      name: `typescript-eslint/${name}`,
```

Comment:
Part of me thinks we should do this as `@` but given they're from the non-namespaced package it's probably better to be consistent and name it without the `@`.